getting_started/scripting/c_sharp/c_sharp_features.rst

```diff
@@ -104,6 +106,38 @@ Emitting signals is done with the ``EmitSignal`` method.
 
 Notice that you can always reference a signal name with the ``nameof`` keyword (applied on the delegate itself).
 
+It is possible to bind values when establishing a connection by passing an object array.
+
+.. code-block:: csharp
+
+    public int Value { get; private set; } = 0;
+
+    private void ModifyValue(int modifier)
+    {
+        Value += modifier;
+    }
+
+    public void SomeFunction()
+    {
+        var plusButton = (Button)GetNode("PlusButton");
+        var minusButton = (Button)GetNode("MunusButton");
+        
+        plusButton.Connect("pressed", this, "ModifyValue", new Object[] { 1 })
```

Comment:
Probably, it's better to use `new object []` here to avoid potential ambiguous reference problem (the argument type here is not `Godot.Object` but `object`) and also to follow the best practice to prefer CLR type names.